src/components/wizard/wizard-step.component.ts

```diff
@@ -20,10 +20,19 @@ export class WizardStepComponent {
     @Input() disableNextWhenInvalid: boolean | undefined;
 
     /** Allows you to define whether or not a step is valid. The user will not be able to proceed to the next step if this property has a value of false. */
-    @Input() valid: boolean = true;
+    _valid: boolean = true;
+
+    @Input()
```

Comment:
Could we add a comment above this input explaining its purpose, it helps editors show a bit of information about the input without users having to consult the documentation